runtime/src/main/java/io/quarkus/vault/runtime/health/VaultHealthCheck.java

```diff
@@ -17,13 +18,22 @@ public class VaultHealthCheck implements HealthCheck {
     @Inject
     VaultClient client;
 
+    @Inject
+    VaultBuildTimeConfig buildTimeConfig;
+
     @Override
     public HealthCheckResponse call() {
 
         final HealthCheckResponseBuilder builder = HealthCheckResponse.named("Vault connection health check");
 
         try {
-            var status = client.sys().health().status()
+            Boolean standbyok = null;
+            Boolean perfstandbyok = null;
```

Comment:
I've chosen nullable objects over primitives because the health client won't even add query parameters if they are passed as `null`.

Reply:
I do not think there is need for this.
those fields had been added for `VaultSystemBackendReactiveEngine.health()`

and in the impl we can see
```
    @Override
    public Uni<VaultHealth> health() {
        boolean isStandByOk = this.buildTimeConfig.health().standByOk();
        boolean isPerfStandByOk = this.buildTimeConfig.health().performanceStandByOk();

        return this.health(isStandByOk, isPerfStandByOk);
    }
```
there is no protection against nullity, and we have defaults for those 2 attributes.
can you check? other than that this looks good.

Reply:
Adapting my code to what already exists in `VaultSystemBackendManager`.